src/app/pages/HomePage/ParaTimeSelector/Graph/index.tsx

```diff
@@ -133,15 +134,51 @@ const GraphCmp: FC<GraphProps> = ({
         fill="white"
       />
-      <circle cx="235.407" cy="36.3673" r="11.8332" transform="rotate(-90 235.407 36.3673)" fill="#01F1E3" />
-      <circle cx="235.335" cy="36.4385" r="8.02878" transform="rotate(-90 235.335 36.4385)" fill="#06152B" />
+      <circle
+        id={`${GraphEndpoint.EMERALD}-tx-outer-circle`}
```

Comment:
Are the IDs just so it's easier to understand SVG code?

Reply:
No, [id attr is used to add styling to elements that behaves like links] (https://github.com/oasisprotocol/explorer/blob/master/src/app/pages/HomePage/ParaTimeSelector/Graph/index.tsx#L54), but this won't be necessary if I will use an anchor.